build.cmd

```diff
@@ -659,6 +659,8 @@ if %__BuildCoreLib% EQU 1 (
             call %__ProjectDir%\dotnet.cmd publish --self-contained -r win-%__BuildArch% -c %__BuildType% -o "%__BinDir%\crossgen2" "%__ProjectDir%\src\tools\crossgen2\crossgen2\crossgen2.csproj"
             copy /Y "%__BinDir%\clrjit.dll" "%__BinDir%\crossgen2\clrjitilc.dll"
             copy /Y "%__BinDir%\jitinterface.dll" "%__BinDir%\crossgen2\jitinterface.dll"
+            echo %__MsgPrefix%Publishing tibcmgr...
+            call %__ProjectDir%\dotnet.cmd publish --self-contained -r win-%__BuildArch% -c %__BuildType% -o "%__BinDir%\tibcmgr" "%__ProjectDir%\src\tools\crossgen2\tibcmgr\tibcmgr.csproj"
```

Comment:
Do we need self-contained publish? Could this build like r2rdump instead? (one needs to provide a matching `dotnet` to run the tool).

We do self-contained publish for crossgen2 because it runs in testing.

Self-contained publish on CoreCLR = 150 MB of junk in the /bin directory that gets replicated to CORE_ROOT and all test machines. I really didn't want to do this for crossgen2, but we don't have other (non-CoreRT) choices.

Reply:
Sure, as you say, this isn't needed by the tests, and its quite expensive to stand a self-contained binary.